CompoundV3-Bots/Reserves-Monitor/src/agent.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  BlockEvent,
+  Finding,
+  HandleBlock,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+} from "forta-agent";
+import CONFIG from "./agent.config";
+import { BigNumber, providers } from "ethers";
+import Fetcher from "./dataFetcher";
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData, abs } from "./utils";
+import { createFinding } from "./finding";
+
+const ALERTS: { [address: string]: number } = {};
+
+const networkManager: NetworkManager<NetworkData> = new NetworkManager(CONFIG);
+const dataFetcher: Fetcher = new Fetcher(getEthersProvider(), networkManager);
+
+export const provideInitialize =
+  (provider: providers.Provider | any) => async () => {
+    const { chainId } = await provider.getNetwork();
+    await networkManager.init(provider);
+    networkManager.setNetwork(chainId);
```

Comment:
```suggestion
    await networkManager.init(provider);
```
No need to do these additional steps if you're already calling `init`.